test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -155,6 +155,46 @@ def test_struct_payload_non_utf8_log
     end
   end
 
+  # For grpc path, the log name is sent as a part of the log entry, thus can be
+  # verified directly.
+  def test_tag_acceptance
+    setup_gce_metadata_stubs
+    [
+      # When require_valid_tags is on, we only convert integer tags to strings.
+      # Other invalid cases will cause the log to be dropped.
+      [123, '123', true],
+      ['test', 'test', true],
+      ['germanß', 'germanß', true],
+      ['chinese中', 'chinese中', true],
+      ['specialCharacter/_-.', 'specialCharacter/_-.', true],
```

Comment:
This line seems outdated. Encoding should have been added in commit d83ef7e already.